src/prompts/view-events-prompt.js

```diff
@@ -1,8 +1,8 @@
 export const VIEW_EVENTS_PROMPT = `
 Date format: YYYY-MM-DDThh:mm:ss+00:00
-Based on this event description: 'View my events today',
+Based on this event description: 'View my events on Saturday',
 output a json of the following parameters: 
-Today's datetime on UTC time 2023-05-02T10:00:00+00:00 and timezone
+Today's datetime on UTC time 2023-05-02T10:00:00+00:00, it's Thursday and timezone
```

Comment:
Looks like 2nd of May 2023 was a Tuesday, should we correct this to avoid risking confusing it?

```suggestion
Today's datetime on UTC time 2023-05-02T10:00:00+00:00, it's Tuesday, and timezone
```